rules/linux/process_creation/proc_creation_lnx_device_driver_insert_remove.yml

```diff
@@ -0,0 +1,28 @@
+title: Detect Adding Kernel Modules - Linux
```

Comment:
This rule is covering both removal and addition so please make the title reflect that